ProjFS.Mac/PrjFSKext/FileSystemMountPoints.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+struct mount;
```

Comment:
I'm not sure that's really necessary. What else would a forward declaration be for?

Reply:
> I'm not sure that's really necessary. What else would a forward declaration be for?

Yeah I agree.  It's been a long time since I left this comment, and I'm not sure why I thought it was worth adding a comment at that time.